### DISCUSSION TITLE
Photoprism web access log

### DISCUSSION THREAD

**Discussion by gitconsumer** [photoprism/photoprism#3513] [General]
Does Photoprism have any kind of web server access log? I am currently investigating a weird hacking issue https://github.com/NginxProxyManager/nginx-proxy-manager/discussions/3036 and starting to think that my photoprism docker container was infected somehow.

**Comment by lastzero** [comment]
https://docs.photoprism.app/getting-started/troubleshooting/docker/#viewing-logs
